src/Security.md

```diff
@@ -20,7 +20,7 @@ You can enable security declaratively or programmatically, as shown below.
 
 ```xml
 <hazelcast xsi:schemaLocation="http://www.hazelcast.com/schema/config
-    http://www.hazelcast.com/schema/config/hazelcast-config-3.3.xsd"
+    https://hazelcast.com/schema/config/hazelcast-config-3.9.xsd"
```

Comment:
In the reference manual we don't make use of https; so from a consistency point of view it is best to keep it at http.